www/front_src/src/Resources/Details/tabs/Graph/index.tsx

```diff
@@ -65,13 +69,19 @@ const GraphTabContent = ({
   return (
     <GraphOptionsContext.Provider value={graphOptions}>
       <div className={classes.container}>
-        <TimePeriodButtonGroup />
-        <MousePositionContext.Provider value={mousePositionProps}>
-          <ExportablePerformanceGraphWithTimeline
-            graphHeight={280}
-            resource={details}
-          />
-        </MousePositionContext.Provider>
+        {details?.type === 'service' ? (
```

Comment:
```suggestion
        {equals(details?.type, 'service') ? (
```
It's more readable to extract it as a variable like `isService`